📝 add docusaurus documentation (#266)
ajnart/homarr: docs/docs/modules/built-in-modules/module-weather.md

---
sidebar_position: 1
tags:
  - Modules
  - Weather
  - Geolocation
  - Forecast
---

# ☔ Weather Module

> The weather module uses your location to display the current, highest and lowest temperature. The module is using [open-meteo.com](https://open-meteo.com) for the retrival of the weather data.

Reviewer: ```suggestion
The Weather Module uses your location to display the current, highest and lowest temperature. The Module uses [open-meteo.com](https://open-meteo.com) to retrieve weather data.
```

> The module **does not request your browsers location**, but you must specify the city in the configuration of the module.

Reviewer: ```suggestion
The Nodule **does not request your browsers location**, you must specify a location yourself using the Module's settings.
```

Reviewer 2: > 

Nodule ?

Reviewer: oops

## Activate the Module
Please read our documentation on [how to enable a module](./../index.md#activating-a-module).

## Configuration

| Configuration         | Description | Values | Default Value |
| --------------------- | ----------- | ------ | ------------- |
| Display in Fahrenheit | Displays the temperature in °F. | yes / no | no |
| Current Location | City name for the weather data retrival | - | - |

![configuration of the weather module](./img/module-weather-configuration.png)

:::tip

> Check out our documentation on [how to configure a module](./../index.md#configure-a-module)

Reviewer: ```suggestion
Check out our documentation on [how to configure a module](./../index.md#configure-a-module).
```
